src/components/library-item/library-item.css

```diff
@@ -91,12 +111,49 @@
     width: 300px;
 }
 
+.featured-extension-text {
+    text-align: left;
+    padding-left: 1.25rem;
+}
+
 .featured-description {
     display: block;
     font-weight: normal;
     line-height: 2;
 }
 
+.featured-extension-metadata {
+    width: 100%;
+    padding: 0rem 1.25rem 1rem 1.25rem;
```

Comment:
Nit: customarily "0" is written with no units in CSS

Reply:
Okay